src/Event/ReadModel/JSONLD/EventLDProjector.php

```diff
@@ -260,7 +260,7 @@ protected function applyEventCreated(
             $jsonLD->creator = $author->toNative();
         }
 
-        $jsonLD->workflowStatus = WorkflowStatus::DRAFT()->getName();
+        $jsonLD->workflowStatus = WorkflowStatus::DRAFT()->toString();
```

Comment:
If I understand correct the old code used the name of the constant which was in uppercase instead of the value which was in lowercase?

Reply:
Correct, in the new class everything is uppercase, so `toString()` works as a replacement.